plsc/entry.py

```diff
@@ -300,20 +311,20 @@ def _get_optimizer(self):
                     values=lr), warmup_steps, start_lr, base_lr)
             else:
                 lr_val = fluid.layers.piecewise_decay(boundaries=bd, values=lr)
-    
+
             optimizer = fluid.optimizer.Momentum(
                 learning_rate=lr_val, momentum=0.9,
                 regularization=fluid.regularizer.L2Decay(5e-4))
             self.optimizer = optimizer
-    
+
         if self.loss_type in ["dist_softmax", "dist_arcface"]:
             self.optimizer = DistributedClassificationOptimizer(
                 self.optimizer, global_batch_size, use_fp16=self.use_fp16,
                 loss_type=self.loss_type,
                 fp16_user_dict=self.fp16_user_dict)
         elif self.use_fp16:
             self.optimizer = fluid.contrib.mixed_precision.decorate(
-                optimizer=optimizer, init_loss_scaling=self.init_loss_scaling)
+                optimizer=optimizer, init_loss_scaling=self.fp16_user_dict['init_loss_scaling'])
```

Comment:
How to pass other parameters, e.g., incr_every_n_steps?

Reply:
done.thanks!